integration-tests/codestarts/src/test/resources/__snapshots__/VaadinExtensionCodestartTest/testApplicationContents/dir-tree.snapshot

```diff
@@ -0,0 +1,7 @@
+/
+themes/
+themes/starter-theme/
+themes/starter-theme/components/
+themes/starter-theme/components/vaadin-text-field.css
+themes/starter-theme/styles.css
+themes/starter-theme/theme.json
```

Comment:
```suggestion
themes/starter-theme/theme.json

```

Reply:
This file is auto-generated by the quarkus tooling and then used in test always by Quarkus code.
So, I think it should not be modified manually

Reply:
oki, thanks!